conda-store-server/conda_store_server/orm.py

```diff
@@ -182,6 +197,14 @@ class Build(Base):
     started_on = Column(DateTime, default=None)
     ended_on = Column(DateTime, default=None)
     deleted_on = Column(DateTime, default=None)
+    build_key_version = Column(Integer, default=_get_build_key_version, nullable=False)
+
+    @validates("build_key_version")
+    def validate_build_key_version(self, key, build_key_version):
+        if build_key_version not in [1, 2]:
+            raise ValueError(f"invalid build_key_version={build_key_version}")
```

Comment:
```suggestion
            raise ValueError(f"invalid build_key_version={build_key_version}. Accepted values are {1, 2}")
```

Proof again that this is missing something. We should be able to easily say which ones are accepted in the error. But we would be forced to hardcode those again.